updating/understanding_updates/intro-to-updates.adoc

```diff
@@ -0,0 +1,44 @@
+:_content-type: ASSEMBLY
+[id="understanding-openshift-updates"]
+= Introduction to OpenShift updates
+include::_attributes/common-attributes.adoc[]
+:context: understanding-openshift-updates
+
+toc::[]
+
+With {product-title} 4, you can update an {product-title} cluster with a single operation by using the web console or the OpenShift CLI (`oc`). Platform administrators can view new update options either by going to *Administration* -> *Cluster Settings* in the web console or by looking at the output of the `oc adm upgrade` command.
+
+Red Hat hosts a public OpenShift Update Service (OSUS) which serves a graph of update possibilities based on the OCP release images in the official registry.
+The graph contains available updates for any public OCP release.
```

Comment:
Let me know if "available updates" can be better reworded, such as changing it to "all supported updates"

Reply:
"available updates" would read well to me, except that it is very close to `availableUpdates`, and `availableUpdates` is (unfortunately in hind sight) the name chosen for currently-recommended-for-this-cluster updates.

Maybe "update recommendations", but folks might interpret that as "recommended" (included in `availableUpdates`) and not realize it also includes "supported but not recommended" (a negative recommendation, in `conditionalUpdates` with `Recommended!=True`).

"Update information" or "update advice" or some such might be both easy-to-think-you-understand and hard-to-misunderstand?

Reply:
Hmm, I think I'll go with "update information" for now, but I'll also save this feedback item for later in case a better option is more obvious when this doc gets refined